### PR TITLE
docs: hacking and cleanup readme

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Add a `HACKING.md` file with some info about the various testnets and light clean up of `README.md`

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Hacking
+
+This document list a couple of useful information to develop NNS-dapp.
+
+## Environments
+
+Testnets [canister_ids.json](https://github.com/dfinity/nns-dapp/blob/testnets/testnets/canister_ids.json) provides an overview of the canister IDs that should be currently deployed and available on various test environments.
```
Comment (lmuntaner) [inline HACKING.md:7]: I would say that this should be copied from here to the local file to be used for local development.

**Review by lmuntaner — APPROVED** [review]
Thanks, just one comment!

File: HACKING.md (hunk 1 above)
Comment (peterpeterparker) [reply HACKING.md:7]: Mmmh I don't get it but, really not against changing it. Can you rephrase the sentence as you wish?

Comment (lmuntaner) [reply HACKING.md:7]: Maybe it should be added below, see my comment there.

File: HACKING.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Configure an environment
+
+If you wish to configure an environment for local development purpose, you can proceed as following:
```
Comment (lmuntaner) [inline HACKING.md:11]: If you wish to work against a testnet, there are two options:

### Script tag

- collect...
- add a new...

### Copy Canister Ids

- Copy the remote canister_ids.json to your local version.
- Change the `DFX_NETWORK=<testnet_name>` in the `dev` script.

Example:
```
"dev": "... DFX_NETWORK=small11 npm run build:config...",
```

**Review by lmuntaner — COMMENTED** [review]
Maybe something like this?

File: HACKING.md (hunk 1 above)
Comment (peterpeterparker) [reply HACKING.md]: Discussed and resolved, thanks for the inputs 👍

File: HACKING.md (hunk 2 above)
Comment (peterpeterparker) [reply HACKING.md]: Discussed and resolved, thanks for the inputs 👍
